openlibrary/templates/covers/add.html

```diff
@@ -68,7 +68,9 @@
             </div>
 
             <div class="formElement" style="margin: $('15px 0px 0px 15px;' if doc.type.key == '/type/work' else '0px;')">
-                <button type="submit" name="upload" id="imageUpload" value="$_('Submit')" class="largest">$_("Submit")</button>
+                <button type="submit" name="upload" id="imageUpload" value="$_('Submit')" class="largest" data-loading-text='$_("Uploading...")'>
```

Comment:
Let's stick to using `data-i18n` for localized strings.

Reply:
Let's chat about it during our ABC call! We use a few different patterns in the code, +1 it would be good to standardize on one.

Reply:
I think I've seen 3 patterns?

1. data-i18n
2. one-offs (like here)
3. IIRC some parts of the codebase have like i18n in a separate <script> tag? I can't recall

It's a touch hard to search for! I think data-i18n is a reasonable standard should we choose to adopt it :)

Reply:
Here are some examples I found of 2/3

2: https://github.com/internetarchive/openlibrary/blob/7fe6b53c4f3e460441391a8aa16a891b7821d923/openlibrary/templates/type/list/view_body.html#L60

3: https://github.com/internetarchive/openlibrary/blob/4ba4f960a17c74b86003632a929ac342c5fef5b0/openlibrary/templates/lists/widget.html#L69